src/server/multi_test.cc

```diff
@@ -727,6 +727,83 @@ TEST_F(MultiTest, TestSquashing) {
   f1.Join();
 }
 
+#if 0
+TEST_F(MultiTest, MultiLeavesTxQueue) {
+  // Tests the scenario, where the OOO multi-tx is scheduled into tx queue and there is another
```

Comment:
Add 
  if (auto mode = absl::GetFlag(FLAGS_multi_exec_mode); mode == Transaction::NON_ATOMIC) {
    GTEST_SKIP() << "Skipped MultiLeavesTxQueue test because multi_exec_mode is non atomic";
    return;
  }